docs/development/translations.rst

```diff
@@ -96,6 +115,25 @@ SecureDrop is translated using the **Weblate** platform. :ref:`Get started using
 
 Sources, journalists and admins use localized versions of SecureDrop. A malicious actor could attempt to modify their behavior by creating misleading translations. In order to mitigate that risk, all translations must be reviewed and accepted by designated :ref:`reviewers <weblate_glossary_reviewer>` before they become part of SecureDrop.
 
+.. _when_is_securedrop_translated:
+
+When does SecureDrop's translation happen?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Most of SecureDrop's components :ref:`components <get_started_using_weblate>`
+are translated during SecureDrop's periodic releases, when the latest
+:ref:`source strings <weblate_glossary_source_string>` are made available for
+translation in Weblate.
+
+The SecureDrop Client can be translated continuously, as new code and source
+strings are merged into the project.  Translations are then finalized during
+the release process.
+
+Both processes are coordinated in collaboration with `Localization Lab
+<https://www.localizationlab.org/>`_.  You can watch for the announcements
```

Comment:
Helpful link to https://www.localizationlab.org/ but should move up to the top with the first mention of the org.

Reply:
As of 6ac2ad8d6c26c0dd6edd7e69cfe066bef85fa588, kept both links for good measure, understanding that folks may enter the documentation at different points.